wp-content/themes/reconasia/assets/_scss/pages/archive.scss

```diff
@@ -82,3 +82,39 @@
     display: none;
   }
 }
+
+.cards__container {
+  @include vw100();
+
+  @include breakpoint('large') {
+    display: flex;
+  }
+}
+
+.card {
+  width: 100%;
```

Comment:
Since these are children of a flex element, I would probably do `flex: 0 0 50%` instead, which says the elements shouldn't grow or shrink, but should each be 50% of their parent.